--- nocc-2003-12-10.orig/lang/en.php	2003-03-03 08:34:53.000000000 +0100
+++ nocc-2003-12-10/lang/en.php	2003-12-13 18:03:25.000000000 +0100
@@ -105,6 +105,11 @@
 $html_filter_from = 'From Field';
 $html_filter_change_tip = "To change a filter simply overwrite it.";
 $html_reapply_filters = "Reapply all filters";
+$html_filter_contains = "contains";
+$html_filter_name = "Filter Name";
+$html_filter_action = "Filter Action";
+$html_filter_moveto = "Move to";
+$html_filter_delete = "Delete";
 
 // Other pages
 $html_select_one = '--Select One--';
