--- nocc-2003-12-10/lang/pl.php.orig	2002-11-29 08:04:57.000000000 +0100
+++ nocc-2003-12-10/lang/pl.php	2003-12-12 13:19:23.000000000 +0100
@@ -10,6 +10,7 @@
  *
  * Configuration file for the Polish language
  * Translation by Ryszard Janiszewski <dex7@akacje.net> 
+ * Translation from 0.9.5 to 0.9.6 by <tommat@pld-linux.pl> 
  */
 
 $charset = 'ISO-8859-2';
@@ -70,6 +71,7 @@
 
 // prefs.php
 
+$html_msgperpage = 'Wiadomo¶ci na stronie';
 $html_preferences = 'Ustawienia';
 $html_full_name = 'Nazwa wy¶wietlana';
 $html_email_address = 'Adres email';
@@ -79,10 +81,46 @@
 $html_reply_to = 'Odpisuj na adres';
 $html_use_signature = 'U¿ywaj podpisu';
 $html_signature = 'Podpis';
+$html_reply_leadin = 'Nag³ówek odpowiedzi';
 $html_prefs_updated = 'Zapisano nowe ustawienia';
+$html_manage_folders_link = 'Zarz±dzaj Foldreami IMAP';
+$html_manage_filters_link = 'Regu³y wiadomo¶ci';
 
-// Other pages
+// folders.php
+$html_folders_create_failed = "Nie mo¿na uworzyæ folderu!";
+$html_folders_sub_failed = "Nie mo¿na do³±czyæ folderu!";
+$html_folders_unsub_failed = "Nie mo¿na od³±czyæ folderu!";
+$html_folders_rename_failed = "Nie mo¿na zmieniæ nazwy folderu!";
+$html_folders_updated = 'Zaktualizowano foldery';
+$html_folder_subscribe = 'Do³±cz';
+$html_folder_rename = 'Zmieñ nazwê';
+$html_folder_create = 'Stwórz nowy folder o nazwie';
+$html_folder_remove = 'Od³±cz folder';
+
+// filters.php
+$html_filter_remove = 'Skasuj';
+$html_filter_body = 'Tre¶æ Wiadomo¶ci';
+$html_filter_subject = 'Temat Wiadomo¶ci';
+$html_filter_to = 'Pole \'Do\'';
+$html_filter_cc = 'Pole \'DW\'';
+$html_filter_from = 'Pole \'Od\'';
+$html_filter_change_tip = "¯eby zmieniæ filtr poprostu go nadpisz.";
+$html_reapply_filters = "Ponownie zastosuj wszystkie filtry";
 
+// Other pages
+$html_select_one = '--Wybierz Jeden--';
+$html_and = 'i';
+$html_new_msg_in = 'Nowe wiadomo¶ci w';
+$html_or = 'lub';
+$html_move = 'Przenie¶';
+$html_copy = 'Skopiuj';
+$html_messages_to = 'zaznaczone wiadomo¶ci do';
+$html_gotopage = 'Przejd¼ do strony';
+$html_gotofolder = 'Przejd¼ do Folderu';
+$html_other_folders = 'Lista Folderów';
+$html_page = 'Strona';
+$html_of = 'z';
+$html_to = 'do';
 $html_view_header = 'Zobacz nag³ówek';
 $html_remove_header = 'Schowaj nag³ówek';
 $html_inbox = 'Skrzynka odbiorcza';
@@ -127,14 +165,15 @@
 $html_low = 'Niska';
 $html_normal = 'Normalna';
 $html_high = 'Wysoka';
+$html_receipt = 'Zarz±daj potwierdzenia otrzymania wiadomo¶ci';
 $html_select = 'Zaznacz';
 $html_select_all = 'Zaznacz wszystko';
 $html_loading_image = '£adowanie obrazka';
 $html_send_confirmed = 'Twoja wiadomo¶æ zosta³a wys³ana';
 $html_no_sendaction = 'Nie okre¶lona akcja. Spróbuj w³±czyæ Javascript.';
-$html_error_occurred = 'An error occurred';
-$html_prefs_file_error = 'Unable to open preferences file for writing.';
-$html_sig_file_error = 'Unable to open signature file for writing.';
+$html_error_occurred = 'Wyst±pi³ b³±d';
+$html_prefs_file_error = 'Nie mo¿na otworzyæ pliku z ustawieniami do zapisu.';
+$html_sig_file_error = 'Nie mo¿na otworzyæ pliku sygnatury do zapisu.';
 
 $original_msg = '--- Wiadomo¶æ oryginalna ---';
 $to_empty = 'Pole \'Do\' nie mo¿e byæ puste!';
@@ -142,8 +181,11 @@
 // SMTP problems (class_smtp.php)
 $html_smtp_error_no_conn = "Nie mo¿na utworzyæ po³±czenia";
 $html_smtp_error_unexpected = "Nieoczekiwana reakcja:";
-$lang_could_not_connect = 'Could not connect to server';  //to translate
+
+// IMAP messages (class_local.php)
+$lang_could_not_connect = 'Nie mo¿na po³aczyæ siê z serwerem';  //to translate
+
 $html_file_upload_attack = 'Possible file upload attack';  //to translate
-$html_invalid_email_address = 'Invalid e-mail address';  //to translate
-$html_seperate_msg_win = 'Messages in separate window';  //to translate
+$html_invalid_email_address = 'Niew³a¶ciwy e-mail adres';
+$html_seperate_msg_win = 'Wiadomo¶ci w osobnym oknie';
 ?>
