--- nocc-2003-12-10.orig/html/filter_prefs.php	2002-12-16 16:23:15.000000000 +0100
+++ nocc-2003-12-10/html/filter_prefs.php	2003-12-13 18:02:18.000000000 +0100
@@ -49,7 +49,7 @@
                         </select>
                     </td>
                     <td align="center" class="prefs" valign="top">
-                        &nbsp;contains&nbsp;
+                        <?php echo $html_filter_contains ?>
                         <td align="left" class="prefs" valign="top">
                             <input type="text" name="contains1" size="20" max="80">
                         </td>
@@ -68,7 +68,7 @@
                         </select>
                     </td>
                     <td align="center" class="prefs" valign="top">
-                        contains
+                        <?php echo $html_filter_contains ?>
                         <td align="left" class="prefs" valign="top">
                             <input type="text" name="contains2" size="20" max="80">
                         </td>
@@ -87,23 +87,23 @@
                         </select>
                     </td>
                     <td align="center" class="prefs" valign="top">
-                        contains
+                        <?php echo $html_filter_contains ?>
                         <td align="left" class="prefs" valign="top">
                             <input type="text" name="contains3" size="20" max="80">
                         </td>
                 </tr>
                 <tr>
                     <td colspan="4" class="prefs" align="center">
-                        Filter Name: <input type="text" name="filtername" size="40" max="80">
+                        <?php echo $html_filter_name ?>: <input type="text" name="filtername" size="40" max="80">
                     </td>
                 </tr>
                 <tr>
                     <td colspan="4" class="prefs" align="center">
-                        Filter Action: 
-                        <input type="radio" name="filter_action" value="MOVE" checked>Move to
+                        <?php echo $html_filter_action ?>: 
+                        <input type="radio" name="filter_action" value="MOVE" checked><?php echo $html_filter_moveto ?>
                         <?php echo $filter_move_to ?>
                         &nbsp;&nbsp;&nbsp;
-                        <input type="radio" name="filter_action" value="DELETE">Delete
+                        <input type="radio" name="filter_action" value="DELETE"><?php echo $html_filter_delete ?>
                     </td>
 
                 <tr>
