--- nocc-1.7/config/conf.php.dist.orig	2009-05-13 09:26:15.661836869 +0200
+++ nocc-1.7/config/conf.php.dist	2009-05-13 09:28:08.515317487 +0200
@@ -195,12 +195,12 @@
 // be readable by everybody, and will contain sensible information as email
 // addresses and names.
 // If left empty, preferences, contacts and session saving will be disabled.
-$conf->prefs_dir = '';
+$conf->prefs_dir = '/var/lib/nocc';
 
 // Default tmp directory (where to store temporary uploaded files)
 // This should be something like '/tmp' on Unix System
 // And 'c:\\temp' on Win32 (note that we must escape "\")
-$conf->tmpdir = '';
+$conf->tmpdir = '/tmp';
 
 // ###
 // ### Optional parameters ###
