--- nocc-2003-12-10/functions.php.orig	2003-03-05 14:28:48.000000000 +0100
+++ nocc-2003-12-10/functions.php	2003-12-11 21:55:07.000000000 +0100
@@ -242,7 +242,7 @@
         'body' => $body,
         'body_mime' => $tmp['mime'],
         'body_transfer' => $tmp['transfer'],
-        'header' => $header,
+	'header' => htmlspecialchars($header),
         'verbose' => $verbose,
         'prev' => $prev_msg,
         'next' => $next_msg,
